src/components/menu-list/menu-list-renderer.tsx

```diff
@@ -242,4 +251,8 @@ export class MenuListRenderer {
 
         return <hr class={classes} />;
     };
+
+    private hasSubItems = (item: MenuItem): boolean => {
```

Comment:
I'd change the commit title to `feat(menu): enable having nested menus (sub-menus)`